hack/charts/cluster-api-operator/templates/infra.yaml

```diff
@@ -38,15 +38,15 @@ metadata:
   annotations:
     "helm.sh/hook": "post-install"
     "helm.sh/hook-weight": "2"
-{{- if or $infrastructureVersion $.Values.configSecret.name }}
+{{- if or $infrastructureVersion $.Values.configSecret.name $.Values.manager }}
```

Comment:
This fixes a bug where `spec:` is not printed when infrastructureVersion .Values.configSecret.name are not specified and Values.manager is.